docs/index.rst

```diff
@@ -697,7 +697,7 @@ The security in place between |sr| and the end clients, e.g. ``appSA``, is as fo
    .. sourcecode:: bash
 
       # First get the KAFKA_CLUSTER_ID
-      KAFKA_CLUSTER_ID=$(curl -s http://localhost:8091/v1/metadata/id | jq -r ".id")
+     KAFKA_CLUSTER_ID=$(curl -s http://localhost:8091/v3/clusters | jq ".data[0].cluster_id"")
```

Comment:
IIUC this won't work due to KREST-19?

Reply:
@rnpridgeon can you please revert so we can merge?

Reply:
seems my local repo and the remote repo diverged causing some funkyness. Did a hard reset on the remote repo and reverted these again. Seems the commit suggested change button maybe more of an annoyance than just commiting the suggestion manually.